HW13/Common/Extensions/WebDriverExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace HW13.Common.Extensions
+{
+    public static class WebDriverExtensions
+    {
+        // extensions method to get WebDriverWait
+        // 'this Iwebdriver driver' shows that this method extends IwebDriver functionality
+        public static WebDriverWait GetWebDriverWait(this IWebDriver driver, int timeoutSeconds = 30, TimeSpan? pollingInterval = null, params Type[] exceptionTypes)
+        {
+            var webDriverWait = new WebDriverWait(driver, TimeSpan.FromSeconds(timeoutSeconds));
+            if (pollingInterval != null) // we can set our custom polling interval (e.g. Timespan.FromSeconds(5)) 
+            {
+                webDriverWait.PollingInterval = (TimeSpan)pollingInterval;
```

Comment:
If you useing nullable types like TimeSpan?
So that means you can use variable value like this pollingInterval.Value
and in IF you can set the following condition pollingInterval.HasValue